native.html

```diff
@@ -12,18 +13,18 @@
         }
     </style>
 </head>
+
 <body>
 
-<div id="root"></div>
+    <div id="root"></div>
 
-<script src="lib/i18n.js"></script>
-<script src="lib/icu-slim.js"></script>
-<script src="node_modules/jquery/dist/jquery.min.js"></script>
-<script src="mathquill/mathquill.js"></script>
-<script src="node_modules/babel-polyfill/browser.js"></script>
```

Comment:
I am nervous about removing this @kevinbarabash. Everything seems to work but there also doesn't seem to be a replacement. Does that seem right to you?